### ISSUE TITLE
Upgrade google-cloud-bigtable 1.6.0 from 0.109.0: BulkMutationBatcher removed

### ISSUE THREAD

**Issue by suztomo** [googleapis/java-bigtable-hbase#2333]
When I tried to upgrade google-cloud-java version to 0.119 (https://github.com/googleapis/java-bigtable-hbase/pull/2332), the compilation failed for BulkMutationGCJClient.java because google-cloud-bigtable 1.6.0 does not have `com.google.cloud.bigtable.data.v2.models.BulkMutationBatcher` any more.

![image](https://user-images.githubusercontent.com/28604/71524747-e4642900-289c-11ea-8a37-bfee95474b7b.png)

Can somebody fix this?

**Comment by suztomo** [comment]
@rahulKQL @kolea2 (got your name from https://github.com/googleapis/google-cloud-java/pull/6309)

**Comment by kolea2** [comment]
Hey @suztomo, we are currently working on upgrading the bigtable dependency in a different branch: https://github.com/googleapis/java-bigtable-hbase/blob/bigtable-client-2.x/pom.xml#L62

**Comment by suztomo** [comment]
Great. I see that `google-cloud-bom:0.116.0-alpha` https://search.maven.org/artifact/com.google.cloud/google-cloud-bom/0.116.0-alpha/pom has 1.5.0. Nice.

```
      <dependency>
        <groupId>com.google.cloud</groupId>
        <artifactId>google-cloud-bigtable</artifactId>
        <version>1.5.0</version><!-- {x-version-update:google-cloud-bigtable:current} -->
      </dependency>
```

Looking forward to the release.

**Comment by igorbernstein2** [comment]
This should be fixed in 1.13.0, which updated javabigtable-client-core to depend on the latest version of java-bigtable

**Comment by suztomo** [comment]
@igorbernstein2 @kolea2 Now Apache Beam uses bigtable-client-core 1.13.0. Thank you for the upgrade and release.
https://github.com/apache/beam/blob/master/buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy#L458
